tests/spec/w3c/defaults-spec.js

```diff
@@ -15,6 +15,7 @@ describe("W3C — Defaults", () => {
       "local-refs-exist": true,
       "check-punctuation": false,
       "check-internal-slots": false,
+      "check-charset": true,
```

Comment:
As above. Set it to flase by default. We will enable it after we test it out a bit.